src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,115 @@
+import React, { useState } from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+import { TodoCondition } from '../../types/TodoCondition';
+import { noop } from '../../utils/noop';
+
+type Props = {
+  todo: Todo,
+  todoCondition: TodoCondition,
+  onDeleteTodo?: (todoId: number) => void,
+  toggleTodo?: (curentTodo: Todo[], status?: boolean | undefined) => void,
+  handleSubmitEditing?: (id: number, editTitle: string) => void,
+};
+
+export const TodoItem: React.FC<Props> = ({
+  todo,
+  todoCondition,
+  onDeleteTodo = noop,
+  toggleTodo = noop,
```

Comment:
why do you need `noop` util function?

Reply:
make your code clearer

Reply:
Not sure, check it out please:
https://github.com/mate-academy/react_todo-app-with-api/pull/455#discussion_r1167570178